src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -65,19 +65,23 @@ class SuggestionProvider extends React.Component {
         .map(file => {
           const isDir = file.type === TYPE_DIRECTORY
           const dirId = isDir ? file._id : file.dir_id
-          let path
+          const urlToFolder = `${window.location.origin}/#/folder/${dirId}`
```

Comment:
`urlToFolder` seems to be the name of a function that transforms an `url` to a `folder`.